config/manager/manager.yaml

```diff
@@ -0,0 +1,96 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    control-plane: controller-manager
+    app.kubernetes.io/name: odf-operator
+    app.kubernetes.io/managed-by: kustomize
+  name: system
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
+  namespace: system
+  labels:
+    control-plane: controller-manager
+    app.kubernetes.io/name: odf-operator
+    app.kubernetes.io/managed-by: kustomize
+spec:
+  selector:
+    matchLabels:
+      control-plane: controller-manager
+      app.kubernetes.io/name: odf-operator-webhook
```

Comment:
why should the operator have webhook as the label value? the app name is `odf-operator` isn't it? iirc pls make sure you aren't editing the selector here if we want the running odf-op be replaces with this as the selector labels are immutable.